transforms/parse5Utils.js

```diff
@@ -53,5 +53,10 @@ export const addClass = (node, className) => {
 export const replaceNode = (node, replacement) => {
   const parentIdx = node.parentNode.childNodes.indexOf(node)
   node.parentNode.childNodes[parentIdx] = replacement
-  replacement.parent = node.parent
+  replacement.parentNode = node.parentNode
```

Comment:
Was this a mistake in the original code or something?

Reply:
Yeah I noticed it in the [original code](https://github.com/salesforce/lwc-codemod/blob/main/transforms/shadowToLight/modifyTemplateHtml.js#L40) and decided to fix it since I was updating the file already.

I don't think it affects anything tho as the tests were passing before and after the change.